p2p/client.go

```diff
@@ -431,6 +526,53 @@ func (c *Client) bootstrapLoop(ctx context.Context) {
 	}
 }
 
+// latest seen height to be retrieved using block-sync
+func (c *Client) setLatestSeenHeight(height uint64) {
+	if height > c.latestSeenHeight {
+		c.latestSeenHeight = height
+	}
+}
+
+// Loop used to request missing blocks via block-sync
+func (c *Client) BlockSyncLoop(ctx context.Context) {
+	c.logger.Info("Started block sync loop.")
+
+	ticker := time.NewTicker(c.conf.BootstrapRetryTime)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case <-ticker.C:
+			c.blocksyncMu.Lock()
+			state, err := c.store.LoadState()
+			if err != nil {
+				continue
+			}
+			for h := state.NextHeight(); h <= c.latestSeenHeight; h++ {
+				_, found := c.heightToSkip[h]
```

Comment:
can you add a comment explaining this

Reply:
comment added

Reply:
I didn't get this sentence

which are the blocks missing to be in sync.